File: eSpaCenter.Services/Database/Recenzija.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eSpaCenter.Services.Database
{
    public class Recenzija
    {
        public int RecenzijaID { get; set; }
        public string? SadrzajRecenzije { get; set; }
        public int Ocjena { get; set; }
        public DateTime DatumKreiranja { get; set; } = DateTime.Now.Date;
        public int KorisnikID { get; set; }

        public virtual Korisnik Korisnik { get; set; } = null!;
    }
}

File: UI/espacenter_mobile/lib/models/recenzija.dart
import 'package:flutter/cupertino.dart';
import 'package:json_annotation/json_annotation.dart';

import 'korisnik.dart';

part 'recenzija.g.dart';

@JsonSerializable()
class Recenzija{
  int? recenzijaID;
  String? sadrzajRecenzije;
  int? ocjena;
  DateTime? datumKreiranja;
  int? korisnikID;
  Korisnik? korisnik;

  Recenzija(this.recenzijaID,this.sadrzajRecenzije,this.ocjena,this.datumKreiranja,this.korisnik,this.korisnikID);

  factory Recenzija.fromJson(Map<String, dynamic> json) => _$RecenzijaFromJson(json);

    Map<String, dynamic> toJson() => _$RecenzijaToJson(this);
}